src/Plugin/ultimate_cron/Scheduler/Crontab.php

```diff
@@ -159,14 +159,15 @@ public function isBehind(CronJob $job) {
     // Check the registered time, and use that if it's available.
     $job_last_ran = $log_entry->start_time;
     if (!$job_last_ran) {
-      $registered = variable_get('ultimate_cron_hooks_registered', array());
-      if (empty($registered[$job->id()])) {
+      $registered = \Drupal::configFactory()->getEditable('ultimate_cron_hooks_registered');
```

Comment:
I have no idea what this was before, but there is no such configuration in 8.x-1.x.